CONTRIBUTING.md

```diff
@@ -8,11 +8,25 @@ While we encourage all contributions, remember that Adamant is used everyday to
 
 ## Creating a Submission
 
-Adamant follows the [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) development process. You should begin by [forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo) this repository. When you have finished making changes on your fork, create a pull request back to this repository.
+Adamant follows the [GitHub flow](https://docs.github.com/en/get-started/quickstart/github-flow) development process. You should begin by [forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo) this repository. When you have finished making changes on your fork, create a [pull request](https://github.com/lasp/adamant/pulls) back to this repository.
 
-Please try to keep submissions small and focused on single fixes, enhancements, or additions. This makes changes easier to review and faster to merge.
+### Pull Request
 
-Once a pull request has been submitted the following will occur:
+Please adhere to the following when submitting a pull request:
+
+- Pull requests should be made against `main`.
+- Keep submissions small and focused on single fixes, enhancements, or additions. This makes changes easier to review and faster to merge.
```

Comment:
It may be less common but is it worth mentioning larger feature cases?

Reply:
My thoughts are to encourage small feature updates. Obviously that won't always be possible, and those cases will have to be managed separately. But those should be the exception, not the rule. I question whether that needs to be addressed directly or not, in light of that.